espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Clear.kt

```diff
@@ -37,4 +40,16 @@ class Clear : RequestHandler<AppiumParams, Void?> {
 
         return null
     }
+
+    @Throws(AppiumException::class)
+    override fun handleCompose(params: AppiumParams): Void? {
```

Comment:
The return type for these methods can be changed to Unit

Reply:
Updated to return type to `Unit` as suggested.